ModularTegustation/tegu_items/gadgets/powered.dm

```diff
@@ -409,16 +423,13 @@
 //Injector
 /obj/item/powered_gadget/enkephalin_injector
 	name = "Prototype Enkephalin Injector"
```

Comment:
Highly suggest giving this range 3 or something, otherwise using melee/gun is 100% better then this item
It dosnt even keep a abnormality stuck to you.

Reply:
Those don't auto set aggro.